gdi/get-data-in/connect/azure/azure-connect.rst

```diff
@@ -114,5 +114,4 @@ After you connect your Azure account to Splunk Observability Cloud, you can do t
 * Import Azure metrics, traces, and metadata. For details about the metrics provided by an Azure integration, see :ref:`azure-metrics`.
 * Learn about Splunk Observability Cloud's :ref:`Azure Infrastructure Monitoring options <infrastructure-azure>`. 
 * Use Splunk Observability Cloud tools to monitor your Azure services, such as :ref:`navigators <use-navigators-imm>` and :ref:`dashboards <azure-dashboards>`.
-* Filter Azure monitoring results using tags or dimensions such as ``region`` and ``host name``. 
```

Comment:
This sentence is confusing - we've been talking about integration filters in the section above but this point regards chart/detector filters